samples/interactive-tutorials/src/main/java/events/ImportUserEventsGcs.java

```diff
@@ -44,81 +42,85 @@ public static void main(String[] args) throws IOException, InterruptedException
-        Operation operation = operationsClient.getOperation(operationName);
-
-        while (!operation.getDone()) {
-          // Keep polling the operation periodically until the import task is done.
-          int awaitDuration = 30000;
-          Thread.sleep(awaitDuration);
-          operation = operationsClient.getOperation(operationName);
-        }
-
-        if (operation.hasMetadata()) {
-          ImportMetadata metadata = operation.getMetadata().unpack(ImportMetadata.class);
-          System.out.printf(
-              "Number of successfully imported events: %s\n", metadata.getSuccessCount());
-          System.out.printf(
-              "Number of failures during the importing: %s\n", metadata.getFailureCount());
-        }
-
-        if (operation.hasResponse()) {
-          ImportUserEventsResponse response =
-              operation.getResponse().unpack(ImportUserEventsResponse.class);
-          System.out.printf("Operation result: %s%n", response);
-        }
-      } catch (InvalidArgumentException e) {
+    try (UserEventServiceClient serviceClient = UserEventServiceClient.create()) {
+      String operationName =
+          serviceClient.importUserEventsCallable().call(importEventsGcsRequest).getName();
+
+      System.out.println("The operation was started.");
+      System.out.printf("OperationName = %s%n", operationName);
+
+      OperationsClient operationsClient = serviceClient.getOperationsClient();
+      Operation operation = operationsClient.getOperation(operationName);
+
+      while (!operation.getDone()) {
+        System.out.println("Please wait till operation is done.");
+        Thread.sleep(30_000);
+        operation = operationsClient.getOperation(operationName);
+      }
+
+      if (operation.hasMetadata()) {
+        ImportMetadata metadata = operation.getMetadata().unpack(ImportMetadata.class);
         System.out.printf(
-            "Given GCS input path was not found. %n%s%n "
-                + "Please run CreateTestResources class to create resources.",
-            e.getMessage());
+            "Number of successfully imported events: %s%n", metadata.getSuccessCount());
+        System.out.printf(
+            "Number of failures during the importing: %s%n", metadata.getFailureCount());
+      } else {
+        System.out.println("Metadata is empty.");
+      }
+
+      if (operation.hasResponse()) {
+        ImportUserEventsResponse response =
+            operation.getResponse().unpack(ImportUserEventsResponse.class);
+        System.out.printf("Operation result: %s%n", response);
+      } else {
+        System.out.println("Operation result is empty.");
       }
-    } catch (BigQueryException e) {
-      System.out.printf("Exception message: %s", e.getMessage());
+    } catch (InvalidArgumentException e) {
+      System.out.printf(
+          "Given GCS input path was not found. %n%s%n"
+              + "Please run CreateTestResources class to create resources.",
```

Comment:
User's typically copy/paste these samples to run. If they need a specific resource, we should explain to them how to make it in a comment at the top of the snippet. 

```suggestion
          "The specified GCS file does not exist. Please make sure to create one following the instructions at the top of this file. ",
```

Reply:
Explanation and all setup instructions will be described in a separate space.
Comment was changed.